### ISSUE TITLE
QTI 3 Player Sandbox - XML Item Rendering Issue

### ISSUE THREAD

**Issue by GiovaneIwamoto** [amp-up-io/qti3-item-player#7]
Hello, I'm experiencing an issue with rendering the XML item located at `qtiv3-examples/packaging/items/associate.xml` in the [QTI 3 Player Sandbox](https://qti.amp-up.io/testrunner/sandbox/). The XML item does not render correctly in the player, and it also does not match the expected display as shown in the [IMS Global QTI v3.0 Implementation Guide](https://www.imsglobal.org/spec/qti/v3p0/impl#h.7cs7637r54vv).

Could you provide any guidance on this rendering issue? Specifically, is there a solution or workaround that would allow the player to properly support and display this type of QTI XML item? Any assistance would be greatly appreciated.

Thank you for your assistance.

![image](https://github.com/user-attachments/assets/e8ec5b67-50a7-4e16-9713-1ae67bce9276)

**Comment by paulgrudnitski** [comment]
The QTI 3 Item Player component does not support (and therefore the Sandbox
does not support) the <qti-associate-interaction> type or the
<qti-graphic-associate-interaction> type because neither of those
interactions are part of QTI 3 Basic (Level 1)or Advanced (Level 2)
conformance certification.

The closest interaction types are <qti-match-interaction>,
<qti-gap-match-interaction>, and <qti-graphic-gap-match-interaction> which *are
*all fully-supported in QTI 3 Item Player.

Let me know if you have any other questions or concerns.  If you really
can't live without Associate interactions then perhaps I would consider
implementing such in 2025.

**Comment by GiovaneIwamoto** [comment]
Thanks for clarifying the lack of support for the associate and graphic associate interaction types in the QTI 3 Item Player, and for explaining the currently supported interaction types. This was really helpful in understanding the compatible options available.

I appreciate your willingness to consider implementing the associate interaction type. If a need arises in the future, I’ll reach out, but there's no requirement for it at the moment.

**Comment by FerreirinhaJean** [comment]
Hello! Is it possible to implement the rendering for Associate interactions?

**Comment by paulgrudnitski** [comment]
> Hello! Is it possible to implement the rendering for Associate interactions?

Yes, it's certainly possible.  Before we evaluate how this might be accomplished, please let us know:

a) Explain your use-case(s), and why one of the 3 variants of Match interaction or 2 variants of Order interaction are insufficient for your measurement use-case(s).

b) If you have example UX's in mind, please share images of them so we can get a clearer picture of what you expect in proper implementations of the various permutations of Associate interaction.
